### PR TITLE
Updating CONTRIBUTING with images for setting up server and remote debugging

### PR DESCRIPTION
**Pull request by NikolasKomonen**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,54 +11,135 @@ vscode-java is a Visual Studio Code extension that uses a language server for it
...
+
+1. Install the required software:
    * latest [Visual Studio Code](https://code.visualstudio.com/)
    * [Node.js](https://nodejs.org/) v4.0.0 or higher
-2. Fork and clone the repository
+2. Fork and clone [this repository](https://github.com/redhat-developer/vscode-java)
+
+  	Keep in mind the final directories will look like:
+	```
+	PROJECT_FOLDER/
```
Comment (fbricon) [inline CONTRIBUTING.md]: this doesn't look properly indented when rendered as markdown. See https://github.com/redhat-developer/vscode-java/blob/b0bdf66cfb7f39e460c709ec9eed397821bc641f/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,54 +11,135 @@ vscode-java is a Visual Studio Code extension that uses a language server for it
...
 This assumes that you are starting on the `vscode-java` directory
 
 1. `cd ..`
 2. Fork and clone the [eclipse.jdt.ls](https://github.com/eclipse/eclipse.jdt.ls) repository
+
+	Ensure the directory looks like:
+	```
+	PROJECT_FOLDER/
+				|--- vscode-java/
+				|--- eclipse.jdt.ls/
```
Comment (fbricon) [inline CONTRIBUTING.md]: this doesn't look properly indented when rendered as markdown. See https://github.com/redhat-developer/vscode-java/blob/b0bdf66cfb7f39e460c709ec9eed397821bc641f/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,54 +11,135 @@ vscode-java is a Visual Studio Code extension that uses a language server for it
...
 You can run faster server builds during development by calling `./node_modules/.bin/gulp dev_server` script instead, this will build server binaries that are required by your host OS only. You can also use `npm run watch-server` which will build and place them on the extension for Java changes. These commands run Maven in offline mode, so you might need to run `build-server` at least once, to fetch all the dependencies.
 
 This will build and place the binaries under the `server` folder. Alternately you can download and use the latest snapshot build from [Eclipse &trade; JDT Language Server](https://github.com/eclipse/eclipse.jdt.ls) project with the following
 
-	```bash
-	$ cd ..\vscode-java
-	$ ./node_modules/.bin/gulp download_server
-	```
+```bash
+$ cd ..\vscode-java
```
Comment (Eskibear) [inline CONTRIBUTING.md]: Should change the slash for consistency.

**Comment by fbricon** [comment]
Thanks @NikolasKomonen !

**Pull request merged**